index.html

```diff
@@ -5,7 +5,7 @@
     <meta charset="utf8" />
     <title>Quiz</title>
 
-    <link href="./public/style.css" rel="stylesheet" />
+    <link href="./public/style.scss" rel="stylesheet" />
```

Comment:
This can't be `.scss` because browser only understand `css`. There should be a command/script on package.json that convert from `.scss` to `.css` while running the `npm run dev`